R/derive_param_confirmed_resp.R

```diff
@@ -0,0 +1,382 @@
+#' Derive Confirmed Response Parameter
+#'
+#' Derive confirmed response parameter
+#'
+#' @param dataset Input dataset
+#'
+#'   The `PARAMCD`, `ADT`, and `AVALC` variables and the variables specified by
+#'   `subject_keys` and `reference_date` are expected.
+#'
+#' @param dataset_adsl ADSL input dataset
+#'
+#'   The variables specified for `subject_keys` are expected. For each subject
+#'   of the specified dataset a new observation is added to the input dataset.
+#'
+#' @param filter_source Source filter
+#'
+#'   All observations in `dataset_source` fulfilling the specified condition are
+#'   considered for deriving the confirmed best overall response.
+#'
+#' @param source_pd Date of first progressive disease (PD)
+#'
+#'   If the parameter is specified, the observations of the input dataset for
+#'   deriving the new parameter are restricted to observations up to the
+#'   specified date. Observations at the specified date are included. For
+#'   subjects without first PD date all observations are take into account.
+#'
+#'   *Permitted Values:* a `date_source` object (see `admiral::date_source()`
```

Comment:
usual admiral:: comment - but you'll probably correct me that this is needed, would help me to understand why